docs/resources.md

```diff
@@ -162,6 +162,60 @@ spec:
       value: gcr.io/staging-images/kritis
 ```
 
+#### Surfacing the image digest built in a task
+
+To surface the image digest in the output of the `taskRun` the builder tool should produce this information in a [oci-layout-image](https://github.com/opencontainers/image-spec/blob/master/image-layout.md) `index.json` file. This file should be placed on a location as specified in the task definition under the resource `outputImagePath`
+
+for example this build-push task defines the `outputImagePath` for the `buildImage` resource in `/worksapce/buildImage`
+```yaml
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: build-push
+spec:
+  inputs:
+    resources:
+    - name: workspace
+      type: git
+  outputs:
+    resources:
+    - name: builtImage
+      type: image
+      outputImagePath: /workspace/builtImage
```

Comment:
interesting, is the `index.json` assumed to be in this folder, i.e. `/workspace/buildImage/index.json` or is `/workspace/buildImage` the complete path to the file?

A couple alternatives that might be a bit more clear:
* calling the config option `outputImageDir`
* Making it so that `outputImagePath` _is_ the complete path, e.g. in this example the user would provide `/workspace/buildImage/index.json` <-- would be the most flexible?

Reply:
since the name of `index.json` should be fixed, I will change the field name to `outputImageDir`